pom.xml

```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.9</version>
+        <version>2.22</version>
```

Comment:
Indeed, and it wasn't visible with normal local builds because I had that content cached, and looked like just another transitory environmental issue with CI.  Solved now though.